beetsplug/export.py

```diff
@@ -78,17 +101,21 @@ def commands(self):
             u'-o', u'--output',
             help=u'path for the output file. If not given, will print the data'
         )
+        cmd.parser.add_option(
+            u'-f', u'--format', default='json',
+            help=u'specify the format of the exported data. Your options are json (deafult), csv, and xml'
+        )
         return [cmd]
 
     def run(self, lib, opts, args):
-
         file_path = opts.output
-        file_format = self.config['default_format'].get(str)
         file_mode = 'a' if opts.append else 'w'
+        file_format = opts.format
```

Comment:
Please preserve the effect of `default_format`.

Reply:
would you prefer I get rid of the `default=json` in...
```
cmd.parser.add_option(
            u'-f', u'--format', default='json',
            help=u"the output format: json (default), csv, or xml"
        )
```
and instead have something like...
```
file_format = opts.format if opts.format else self.config['default_format'].get(str)
```